homeassistant/components/zha/light.py

```diff
@@ -384,6 +394,6 @@ def async_restore_last_state(self, last_state):
                 if color_loop_active == 1:
                     self._effect = light.EFFECT_COLORLOOP
 
-    async def refresh(self, time):
+    async def _refresh(self, time):
         """Call async_get_state at an interval."""
         await self.async_get_state(from_cache=False)
```

Comment:
Where are we scheduling an entity state update? I can't find that.

Reply:
Fixed. Great catch!